ResearchKit/Consent/ORKConsentSection.m

```diff
@@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
         ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
+        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
```

Comment:
Not sure where the JSON you are talking about is.
Can you point me toward it

J

Sent from my iPhone 6plus

> On May 22, 2015, at 1:30 PM, jwe-apple <notifications@github.com> wrote:
> 
> In ResearchKit/Consent/ORKConsentSection.m:
> 
> > @@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
> >          ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
> >          ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
> >          ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
> > +        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
> You will also need to update the JSON serialization, ORKESerialization and make sure the tests pass in the ORKTest app.
> 
> —
> Reply to this email directly or view it on GitHub.
>